layer/nrlf/core/authoriser.py

```diff
@@ -53,3 +54,30 @@ def get_pointer_types(
             error=str(exc),
         )
         raise exc
+
+
+def parse_permissions_file(connection_metadata: ConnectionMetadata) -> list[str]:
+    ods_code = ".".join(connection_metadata.ods_code)
+
+    app_id = connection_metadata.client_rp_details.developer_app_id
+    ods_code = connection_metadata.ods_code
+    ods_code_extension = connection_metadata.ods_code_extension
+
+    if ods_code_extension:
+        key = f"{app_id}/{ods_code}.{ods_code_extension}.json"
```

Comment:
This is ok, but it would be subject to change when we do NRL-766. So maybe a comment to remember would be good to have or a comment in the jira.